_posts/2024-06-27-WildFly-playing-with-generative-ai.adoc

```diff
@@ -0,0 +1,319 @@
+---
+layout: post
+title:  "Playing with Generative AI with WildFly"
+date:   2024-06-27
+tags: generative-ai rag wildfly
+author: ehugonne
+description: How to try the Generative AI feature pack with WildFly
+---
+
+This blog post provides information on how to use the AI Galleon Feature pack to write generative AI applications.
+Please note that this feature pack is currently a proof of concept and might not be ready for production.
+
+
+== What's in this feature pack ?
+
+This feature pack is a work in progress so we will only cover what is already available as of today.
+It relies on configuring and exposing https://github.com/langchain4j[LangChain4J] to you application.
+It provides a way to inject some basic elements to make creating a Retrieval-Augmented Generation (RAG) application easy.
+It helps by configuring and providing all the required elements for such an application:
```

Comment:
s/application:/application./g

unless you intended to add a list of required elements